src/components/directory-content.js

```diff
@@ -512,11 +520,38 @@ const DirectoryContent = () => {
         return href;
     }
 
+    function buildTypeWithSubtype(type, subtype) {
+        switch (type) {
+            case elementType.FILTER:
+                return subtype === filterSubtype.SCRIPT
+                    ? filterSubtype.SCRIPT
+                    : type;
```

Comment:
replace `type` by `filterSubtype.FILTER` to be clearer

Reply:
done